.travis.yml

```diff
@@ -19,6 +19,6 @@ install:
   - make install
 script:
   - make lint
-  - QIIMETEST= make test-cov
+  - make test
   - QIIMETEST= source tab-qiime
-  - QIIMETEST= qiime
```

Comment:
Running `qiime info` instead of `qiime` has the opportunity to fail b/c it does more work, plus the command gives some useful debug info in the Travis-CI log.